addons/explosives/XEH_PREP.hpp

```diff
@@ -37,3 +37,5 @@ PREP(spawnFlare);
 PREP(startDefuse);
 PREP(startTimer);
 PREP(triggerType);
+PREP(allowDefuse);
+PREP(isAllowedDefuse);
```

Comment:
```suggestion
PREP(isAllowedDefuse);

```
You should get EditorConfig for your editor, it will do this automatically. :)

Reply:
Never realized it needed a trailing line, is that just ACE convention? Can't remember reading that on the page about it..